apps/src/turtle/blocks.js

```diff
@@ -1129,6 +1129,39 @@ exports.install = function (blockly, blockInstallOptions) {
         '", null, \'block_id_' + this.id + '\');\n';
   };
 
+  blockly.Blocks.turtle_sticker_with_size = blockly.Blocks.turtle_stamp = {
+    helpUrl: '',
+    init: function () {
+      this.setHSV(184, 1.00, 0.74);
+      var dropdown;
+      var input = this.appendDummyInput();
+      input.appendTitle(msg.setStickerSize());
+
+      // Generates a list of pairs of the form [[url, name]]
+      var values = [];
+      for (var name in skin.stickers) {
+        var url = skin.stickers[name];
+        values.push([url, name]);
+      }
+
+      dropdown = new blockly.FieldImageDropdown(values, 40, 40);
+
+      input.appendTitle(dropdown, 'VALUE');
```

Comment:
I wonder if we can share the code from line 1134-1149 and 1152-1155 with `blockly.Blocks.sticker` above?

Reply:
Deleted "turtle_stamp" from lines 1132 & 1160.